float32 roll
float32 pitch
float32 yaw
uint8 coord_type
